app/views/groups/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<section class="main-section">
+<h1 class="header">NEW CATEGORY</h1>
+<%= render "form", group: @group %>
+<div class="links-wrapper"> <%= link_to "Back to Categories", groups_path, class:"link" %></div>
+</section>
```

Comment:
- [ ] Great work maintaining the colours from the heading as requested by the previous reviewer. You can make the design of the form inputs on both Groups and Transaction pages look more identical to that of the design template as requested by the previous reviewer. The form inputs that you have on these pages are basic and don't look identical to that of the design template. Kindly make this adjustment. 👍 😎 

| Your Design - Groups Form  | Your Design - Transaction Form | Design Template Form|
| ------------------------------- | ---------------------------------- | ------------------------ |
| ![image](https://user-images.githubusercontent.com/77106962/235504892-2ddbf499-58e5-4d26-9661-c93c06aa8883.png) | ![image](https://user-images.githubusercontent.com/77106962/235504618-b9999e13-d623-4bb6-b814-4edab2fd9fec.png) | ![image](https://user-images.githubusercontent.com/77106962/235505061-7ccae20e-0870-42db-9e37-ee86dfdc6b50.png) |